validator-ui/src/pages/homepage.jsx

```diff
@@ -32,7 +78,34 @@ export default function Homepage() {
           },
         ]}
       />
-      <div className="flex flex-col items-center justify-center h-screen"></div>
+      <div className="h-screen m-10">
+        <div className="flex justify-between flex-wrap">
+          <div>
+            <h1 className="text-4xl">Companii</h1>
+            <p className="font-semibold">600 de rezultate</p>
+          </div>
+          <div className="hidden md:block">
+            <label for="sort" className="mr-4">
+              Sorteaza
+            </label>
+            <select
+              name="sort"
+              className="bg-bg-header rounded-sm px-2 py-2 outline-none text-sm"
+            >
+              <option value="">Dupa numarul de joburi active</option>
+              <option value="">Dupa numarul de joburi active 2</option>
+              <option value="">Dupa numarul de joburi active 1</option>
+              <option value="">Dupa numarul de joburi active 3</option>
+            </select>
+          </div>
+        </div>
+
+        <div className="flex flex-wrap gap-4 gap-y-10 mt-10 justify-center md:justify-normal">
```

Comment:
schimba  ` md:justify-normal` in  `justify-center` pentru a centra cardurile pe centru
```suggestion
        <div className="flex flex-wrap gap-4 gap-y-10 mt-10 justify-center justify-center">
```